client/src/assets/shrug-emoticon.svg

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should minify this svg

Reply:
i already used https://www.svgminify.com 🤷🏻‍♀️